documentation/1.1/roadmap.md

```diff
@@ -30,15 +30,15 @@ However, our roadmap is quite well-defined:
 <div id="milestones-progress">
     <div id="milestone-overall">Loading…</div>
     <h3>Detail</h3>
-    <div data-title="Typechecker / language specification" data-milestone="https://api.github.com/repos/ceylon/ceylon-spec/milestones/11?callback=?">Loading…</div>
-    <div data-title="JVM compiler / documentation compiler" data-milestone="https://api.github.com/repos/ceylon/ceylon-compiler/milestones/11?callback=?">Loading…</div>
-    <div data-title="JS compiler" data-milestone="https://api.github.com/repos/ceylon/ceylon-js/milestones/8?callback=?">Loading…</div>
-    <div data-title="Language module" data-milestone="https://api.github.com/repos/ceylon/ceylon.language/milestones/10?callback=?">Loading…</div>
-    <div data-title="Module resolver" data-milestone="https://api.github.com/repos/ceylon/ceylon-module-resolver/milestones/10?callback=?">Loading…</div>
-    <div data-title="Runtime" data-milestone="https://api.github.com/repos/ceylon/ceylon-runtime/milestones/11?callback=?">Loading…</div>
-    <div data-title="Common" data-milestone="https://api.github.com/repos/ceylon/ceylon-common/milestones/7?callback=?">Loading…</div>
-    <div data-title="IDE" data-milestone="https://api.github.com/repos/ceylon/ceylon-ide-eclipse/milestones/11?callback=?">Loading…</div>
-    <div data-title="SDK" data-milestone="https://api.github.com/repos/ceylon/ceylon-sdk/milestones/8?callback=?">Loading…</div>
-    <div data-title="Formatter" data-milestone="https://api.github.com/repos/ceylon/ceylon.formatter/milestones/6?callback=?">Loading…</div>
+    <div data-title="Typechecker / language specification" data-repo="ceylon-spec" data-milestone="11">Loading…</div>
+    <div data-title="JVM compiler / documentation compiler" data-repo="ceylon-compiler" data-milestone="11">Loading…</div>
+    <div data-title="JS compiler" data-repo="ceylon-js" data-milestone="8">Loading…</div>
+    <div data-title="Language module" data-repo="ceylon.language" data-milestone="10">Loading…</div>
+    <div data-title="Module resolver" data-repo="ceylon-module-resolver" data-milestone="10">Loading…</div>
+    <div data-title="Runtime" data-repo="ceylon-runtime" data-milestone="11">Loading…</div>
+    <div data-title="Common" data-repo="ceylon-common" data-milestone="7">Loading…</div>
+    <div data-title="IDE" data-repo="ceylon-ide-eclipse" data-milestone="11">Loading…</div>
+    <div data-title="SDK" data-repo="ceylon-sdk" data-milestone="8">Loading…</div>
+    <div data-title="Formatter" data-repo="ceylon.formatter" data-milestone="6">Loading…</div>
```

Comment:
This change isn’t really necessary – we could regexp the milestone number and repo name out of the URL. However, I feel it’s cleaner this way.

(For the record, the regex is
```
data-milestone="https://api\.github\.com/repos/ceylon/\([^/]*\)/milestones/\([[:digit:]]*\)\?callback=\?"
→
data-repo="\1" data-milestone="\2"
```
in Emacs syntax.)